packages/openapi-react-query/src/index.ts

```diff
@@ -121,7 +121,11 @@ export default function createClient<Paths extends {}, Media extends MediaType =
     useQuery: (method, path, ...[init, options, queryClient]) =>
       useQuery(queryOptions(method, path, init as InitWithUnknowns<typeof init>, options), queryClient),
     useSuspenseQuery: (method, path, ...[init, options, queryClient]) =>
-      useSuspenseQuery(queryOptions(method, path, init as InitWithUnknowns<typeof init>, options), queryClient),
+      useSuspenseQuery(
+        // @ts-expect-error TODO: fix minor type mismatch between useQuery and useSuspenseQuery
```

Comment:
cc @kerwanp there’s an error on the newer version of `@tanstack/react-query` here, but it looks like a superficial one:

```
Argument of type 'UseQueryOptions<Response["data"], Response["error"], Response["data"], QueryKey<Paths, Method, Path>>' is not assignable to parameter of type 'UseSuspenseQueryOptions<Response["data"], Response["error"], Response["data"], QueryKey<Paths, Method, Path>>'.
  Types of property 'queryFn' are incompatible.
    Type 'unique symbol | QueryFunction<Response["data"], QueryKey<Paths, Method, Path>, never> | undefined' is not assignable to type 'QueryFunction<Response["data"], QueryKey<Paths, Method, Path>, never> | undefined'.
      Type 'typeof skipToken' is not assignable to type 'QueryFunction<Response["data"], QueryKey<Paths, Method, Path>, never>'.ts(2345)
```

I don’t think it actually affects runtime too much. And the solution may be hairy / didn’t want to destroy the inference happening. So just ignored it for now.